libfwupdplugin/fu-dpaux-device.c

```diff
@@ -72,7 +74,20 @@ fu_dpaux_device_probe(FuDevice *device, GError **error)
 		    g_path_get_basename(fu_udev_device_get_sysfs_path(FU_UDEV_DEVICE(device)));
 		fu_device_set_logical_id(device, logical_id);
 	}
-	return fu_udev_device_set_physical_id(FU_UDEV_DEVICE(device), "pci,drm_dp_aux_dev", error);
+
+	if (!fu_udev_device_set_physical_id(FU_UDEV_DEVICE(device), "pci,drm_dp_aux_dev", error))
+		return FALSE;
+
+	/* only populated on real system, test suite won't have udev_device set */
+	tmp = fu_udev_device_get_sysfs_attr(FU_UDEV_DEVICE(device), "name", NULL);
+	if (tmp != NULL)
+		fu_device_set_name(device, tmp);
```

Comment:
i'm confused why this is needed -- `fu_udev_device_probe()` has:

	if (g_strcmp0(priv->subsystem, "drm_dp_aux_dev") == 0) {
		tmp = g_udev_device_get_sysfs_attr(priv->udev_device, "name");
		if (tmp != NULL && fu_device_get_name(device) == NULL)
			fu_device_set_name(device, tmp);
	}

Reply:
Ignore me, I see that this patch removed it from the baseclass now.